PULL_REQUEST_TEMPLATE.MD

```diff
@@ -1,3 +1,5 @@
+[Replace this line with a breif description of the work done or expected outcome]
```

Comment:
Spell check... :-)

Reply:
I would make a "Description" header and then put a placeholder below describing the expectations

Reply:
Sadly it seems like visual studio code doesn't flag on that and I missed the transposition.  I'll get it fixed.
<img width="687" alt="image" src="https://user-images.githubusercontent.com/87203006/155610510-578bb6b1-9743-4c50-a2f5-483d40127a9a.png">